pkg/ebpf/tracee.go

```diff
@@ -1346,6 +1355,126 @@ func (t *Tracee) initBPF() error {
 	return errfmt.WrapError(err)
 }
 
+func clearMap(bpfMap *bpf.BPFMap) error {
```

Comment:
To better organize code, we should place event specific code (hidden modules) in its own file and not under tracee.go